runtime/lua/vim/lsp/codelens.lua

```diff
@@ -48,7 +48,7 @@ local function execute_lens(lens, bufnr, client_id)
 
   local client = vim.lsp.get_client_by_id(client_id)
   assert(client, 'Client is required to execute lens, client_id=' .. client_id)
-  client:_exec_cmd(lens.command, { bufnr = bufnr }, function(...)
+  client:exec_cmd(lens.command, { bufnr = bufnr }, function(...)
     vim.lsp.handlers[ms.workspace_executeCommand](...)
     M.refresh()
   end)
```

Comment:
@mfussenegger the handler here looks redundant to me. wdyt?

Reply:
I'm not sure but on a first glance it doesn't look like it to me.

Some code lens commands can update the buffer and influence how the code lens itself is displayed. For example in `hls` you can have `Evaluate` code-lens and once you execute it, it evaluates a doctest expression, inserts the result and the codelens title changes to `Refresh`

Reply:
Right, so the handler is just needed for the refresh? If so, then there's a small bug here in that the handler should look for a client specific handler and call that if there is one.

Reply:
Yes, it's only to add the refresh - and good call, it should lookup client.handlers too